core/src/avm2.rs

```diff
@@ -66,7 +66,8 @@ mod vtable;
 pub use crate::avm2::activation::Activation;
 pub use crate::avm2::array::ArrayStorage;
 pub use crate::avm2::call_stack::{CallNode, CallStack};
-pub use crate::avm2::domain::Domain;
+#[allow(unused)] // For debug_ui
+pub use crate::avm2::domain::{Domain, DomainPtr};
```

Comment:
I'm not seeing any actual `DomainPtr` usages added to this file. Is this a leftover, hidden by the `allow`?

Reply:
It's used by `handle.rs` through the re-export

Reply:
Ah, okay. I won't block on this then.